packages/lesswrong/components/comments/CommentsListSection.tsx

```diff
@@ -142,20 +144,29 @@ const CommentsListSection = ({
     setAnchorEl(null);
   }
 
+  const [restoreScrollPos, setRestoreScrollPos] = useState(-1);
+
+  useEffect(() => {
+    if (restoreScrollPos === -1) return;
+
+    window.scrollTo({top: restoreScrollPos})
+    setRestoreScrollPos(-1);
+  }, [restoreScrollPos])
```

Comment:
not a change request but a musing.

this can be 

```typescript
  const setRestoreScrollPos = async (pos: number) => {
       setTimeout(() => {
      window.scrollTo({top: pos})
    }, 0)
  }
```
without a need for useeffecat, but I think it works tiny bit worse

Reply:
Huh, I tried it that way at first, and tbh I went with useEffect just because it was more React-like. I do like how setTimeout is simpler. I have no strong opinion either way.